tests/jballerina-unit-test/src/test/resources/test-src/javainterop/basic/field_access_mutate_tests.bal

```diff
@@ -16,6 +17,19 @@ function testStaticPrimitiveFieldMutate(int shortValue) {
     return setShortValue(shortValue);
 }
 
+function testStaticBooleanFieldMutate() {
+    boolean bool = true;
+    setBoolean(bool);
+    test:assertEquals(getBoolean(), true);
+}
+
+public type bool true|false|5;
```

Comment:
It is a good practice to declare the variable before its use for vertical clarity.